### PR TITLE
Add getMarkets method

### PR DESCRIPTION
**Pull request by youaresofunny**
Simple http method added
Related issue: https://github.com/binance-chain/javascript-sdk/issues/78

### CODE REVIEW

File: __tests__/client.test.js [hunk 1]
```diff
@@ -186,4 +186,12 @@ describe("BncClient test", async () => {
     const res1 = client.createAccountWithKeystore("12345678")
     expect(res1.address.includes('bnb')).toBeTruthy()
   })
+
+  it("get markets works", async () => {
+    const client = await getClient(false)
+    const { result: markets, status } = await client.getMarkets(150)
+
+    expect(markets.length).toBe(150)
```
Comment (notatestuser) [inline __tests__/client.test.js]: This test could fail if the number of available markets is less than 150. 

I would recommend checking that `length > 0` and that the first market contains expected fields (`base_asset_symbol`, `quote_asset_symbol`, `price`, etc).

File: src/client/index.js [hunk 2]
```diff
@@ -449,6 +450,21 @@ export class BncClient {
...
+   * @param {Number} offset from beggining, default 0
+   * @param {Number} limit, max 1000 is default
+   * @return {Object} http response
+   */
+  async getMarkets(limit=1000, offset=0) {
+    try {
+      const data = await this._httpClient.request("get", `${api.getMarkets}?limit=${limit}&offset=${offset}`)
+      return data
+    } catch(err) {
+      return []
```
Comment (notatestuser) [inline src/client/index.js]: It should probably do _something_ with the error, maybe just log it for now.

**Comment by youaresofunny** [comment]
@notatestuser updated

**Comment by notatestuser** [comment]
Looks good, thank you!

**Pull request merged**